info.go

```diff
@@ -259,6 +262,15 @@ func (pi *ProgramInfo) Runtime() (time.Duration, bool) {
 	return time.Duration(0), false
 }
 
+// RecursionMisses returns the total number of times the program was NOT called.
```

Comment:
What could lead to a program not being called? Sounds important to briefly call out here.